src/screens/CreatePost/index.js

```diff
@@ -407,7 +407,7 @@ const CreatePost = () => {
   };
 
   const navigateToTopicPage = () => {
-    return navigation.navigate('TopicPageScreen', {id: initialTopic[0]});
+    return navigation.replace('TopicPageScreen', {id: initialTopic[0]});
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `navigation.navigate` to `navigation.replace` might have implications on the navigation stack. The `replace` method will replace the current screen in the stack with the new one, whereas `navigate` would push a new screen onto the stack. If there's a requirement for the user to go back to the previous screen, this change could cause issues.
```diff
- return navigation.replace('TopicPageScreen', {id: initialTopic[0]});
+ return navigation.navigate('TopicPageScreen', {id: initialTopic[0]});
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->